### ISSUE TITLE
Bard's Tale 1, The - Tales Of The Unknown (1987)

### ISSUE THREAD

**Issue by TimberlAndRE-de** [exoscoriae/eXoDOS#2241]
I found an extra:

- [x] Hintbook from 1986

added to the github

**Comment by exoscoriae** [comment]
added

**Comment by Eulisker** [comment]
I found a nicer looking 

- [ ] Command Summary Card

added to the github folder.

**Comment by exoscoriae** [comment]
replaced

**Comment by TimberlAndRE-de** [comment]
wysiwyg found

- [ ] clue book offer insert (it's from a later re-release collection and for Bard's Tale II and III too, could also be added there)
- [ ] command summary (quality upgrade)
- [ ] disk switch offer insert

all added to the github folder

**Comment by exoscoriae** [comment]
added

**Comment by TimberlAndRE-de** [comment]
found

- [ ] novels (1-5)

**Comment by exoscoriae** [comment]
they suck quality. would be great to find pdfs or scans.

I found a replacement for book 4. I also found book 5 was actually book 7.

True list is:
Tales of the Unknown Book 1 - Castle Of Deception (1992)
Tales of the Unknown Book 2 - Fortress of Frost and Fire (1993)
Tales of the Unknown Book 3 - Prison of Souls (1993)
Tales of the Unknown Book 4 - The Chaos Gate (1994)
Tales of the Unknown Book 5 - Thunder of the Captains (1996)
Tales of the Unknown Book 6 - Wrath of the Princes (1997)
Tales of the Unknown Book 7 - Escape from Roksamur (1997)
Tales of the Unknown Book 8 - Curse of the Black Heron (1998)

Books 5, 6, and 8 are completely missing currently.
Books 1, 2, 3, and 7 are in a very low quality ePub format with no formatting.
